.github/workflows/ci.yml

```diff
@@ -85,6 +85,41 @@ jobs:
         chmod +x codecov
         ./codecov
 
+  #
+  # Android build job
+  #
+  android:
+    runs-on: macos-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+
+    # Definition of the build matrix
+    strategy:
+      matrix:
+        java: [ 11 ]
+        entry:
+          - { mock-maker: 'mock-maker-default', member-accessor: 'member-accessor-default' }
+
+    # All build steps
+    steps:
+    - name: 1. Check out code
+      uses: actions/checkout@v3 # https://github.com/actions/checkout
+      with:
+        fetch-depth: '0' # https://github.com/shipkit/shipkit-changelog#fetch-depth-on-ci
+
+    - name: 2. Set up Java ${{ matrix.java }}
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'zulu'
+        java-version: ${{ matrix.java }}
+
+    - name: 3. Run Android tests on Java ${{ matrix.java }} with ${{ matrix.entry.mock-maker }} and ${{ matrix.entry.member-accessor }}
+      uses: reactivecircus/android-emulator-runner@v2
+      with:
+        api-level: 29
+        script: ./gradlew :androidTest:connectedCheck --no-daemon --no-build-cache
```

Comment:
Upload resulting report as artifact?

Reply:
That's a good question for @TimvdLippe , I don't see test reports uploaded (but only coverage reports).

Reply:
😲 so how do you know what failed?
https://github.com/mockito/mockito/actions/runs/1865554496/jobs/2625339758

Reply:
You look into Github action output, like https://github.com/mockito/mockito/pull/2893#issuecomment-1410906236

Reply:
Hmm... that leaves so many questions open:
```
java.lang.NoClassDefFoundError: org.mockito.internal.debugging.LocationImpl
at org.mockito.internal.debugging.LocationFactory.create(LocationFactory.java:17)
```
is a very minimal output, where is it called from? which code path got here? what's the root "caused by"? What got output to stdout/stderr? How long did it take? etc. All of these are recorded in the report. As an example open any `IntgTest` from [this workflow's "Test Results HTML" artifacts](https://github.com/TWiStErRob/net.twisterrob.gradle/actions/runs/4052565235). Anyway up to you if like reading logs :), it would drive me crazy.

Reply:
:D @TWiStErRob I am totally in for test report publishing (don't get me wrong), I have just included the most interesting part but here is the complete log [1], this is how the test failures are troubleshot at the moment. 

[1] https://github.com/reta/mockito/actions/runs/4057324609/jobs/6982933109

Reply:
We don't have test reporting setup. For that, we rely on console output solely and locally reproducing the error. With CI that is sometimes tricky, but that's the setup we got atm. If you have any suggestions, feel free to create an issue and we can discuss further there.